src/pages/task/taskTabs/buildBaronAndAnnotations/JiraTicketList/JiraTicketRow/index.tsx

```diff
@@ -0,0 +1,82 @@
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import { palette } from "@leafygreen-ui/palette";
+import { Disclaimer } from "@leafygreen-ui/typography";
+import { useAnnotationAnalytics } from "analytics";
+import { StyledLink } from "components/styles";
+import { getJiraTicketUrl } from "constants/externalResources";
+import { size } from "constants/tokens";
+import { TicketFields } from "gql/generated/types";
+import { useSpruceConfig, useDateFormat } from "hooks";
+import { trimStringFromMiddle } from "utils/string";
+
+const { gray } = palette;
+
+interface JiraTicketRowProps {
+  jiraKey: string;
+  fields: TicketFields;
+}
+const JiraTicketRow: React.FC<JiraTicketRowProps> = ({ fields, jiraKey }) => {
+  const annotationAnalytics = useAnnotationAnalytics();
+  const getDateCopy = useDateFormat();
+  const spruceConfig = useSpruceConfig();
+  const jiraHost = spruceConfig?.jira?.host;
+  const url = getJiraTicketUrl(jiraHost, jiraKey);
+  const { assigneeDisplayName, created, status, summary, updated } =
+    fields ?? {};
+  return (
+    <Container data-cy="jira-ticket-row">
+      <JiraSummaryLink
+        href={url}
+        data-cy={jiraKey}
+        onClick={() =>
+          annotationAnalytics.sendEvent({ name: "Click Jira Summary Link" })
+        }
+        title={summary}
+      >
+        {jiraKey}: {trimStringFromMiddle(summary, 80)} {"   "}
```

Comment:
Since the spacing is managed via grid we don't need to add spaces
```suggestion
        {jiraKey}: {trimStringFromMiddle(summary, 80)}
```